main.tf

```
@@ -0,0 +1,99 @@
+module "label" {
+  source      = "git::https://github.com/cloudposse/terraform-null-label.git?ref=master"
```

```suggestion
  source      = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.7.0"
```

always pin to a release to avoid breaking changes in the master branch.
we pin to `master` in examples for two reasons:

1. Show people that they need to pin it 
2. We don't have to update all the examples in all modules (too much work)

---

I did it because I saw examples of it in other repos. I also agree that it has to be pinned to a certain version.